### ISSUE TITLE
Проблема со сборкой конфигурации. Ошибка платформы.

### ISSUE THREAD

**Issue by Pr-Mex** [xDrivenDevelopment/v83unpack#13]
Проверялось на 8.3.4.465 и 8.3.5.1119.
Проблема следующая.
При загрузке cf  из XML в ПАКЕТНОМ режиме в только что созданную базу, некорректно проставляются галочки в ролях.  Проблема воспроизводится стабильно, вне зависимости от того какой режим совместимости выбран в загружаемой конфигурации.
Если делать загрузку через меню интерфейса конфигуратора - то ошибки не возникает.
Опытным путём выяснено, что решить проблему помогает повторная загрузка в пакетном режиме этих же xml в ту же базу.

Т.к. загрузка cf из xml бывает довольно долгой, предлагается на шаге, перед загрузкой из xml вставить промежуточный шаг. На этом шаге будет загружен только Configuration.xml, без объектов из дерева конфигурации. Это не должно занять много времени. После этого загрузка из xml проходит корректно.

**Comment by ghost** [comment]
@Pr-Mex  Я смотрю ты баг в платформе зарегистрировал ? 
- Код ошибки: 30010599
  ----------------------------------------
  Добрый день.
  Технологическая платформа 8.3.5.1146
  При загрузке конфигурации в пакетном режиме из файлов XML не корректно загружаются роли.

**Comment by Pr-Mex** [comment]
Да. На https://bugboard.v8.1c.ru  она ещё не появилась, но 1С ошибку с загрузкой ролей в пакетном режиме признал.

**Comment by Pr-Mex** [comment]
Код ошибки 30010599.
Проверил на 8.3.5.1218 - ничего не изменилось.

**Comment by artbear** [comment]
@Pr-Mex Проверишь на 8.3.5.1248 ?

**Comment by Pr-Mex** [comment]
Проверил. Ничего не изменилось.

**Comment by JohnyDeath** [comment]
1С пишет, что исправили в 8.3.6.1760 https://bugboard.v8.1c.ru/error/000002112.html
На текущий момент доступна для скачивания: 8.3.6.1848 
Не проверял?

**Comment by Pr-Mex** [comment]
Да. Проверял. Исправлено.

**Comment by karnilaev** [comment]
В версии платформы 8.3.7.1759 опять возник баг с установкой неверных флагов в ролях и, к сожалению, повторная загрузка тех же самых файлов XML не помогает.
![Права в хранилище 1С](https://cloud.githubusercontent.com/assets/5828648/11380886/e2aeb7d8-9309-11e5-97bf-e472989ea9a7.png)
![Права в конфигурации после загрузки из файлов XML](https://cloud.githubusercontent.com/assets/5828648/11380890/e2edbfa0-9309-11e5-954e-437cd4a4e47a.png)

Это только у меня так?

**Comment by karnilaev** [comment]
Хотя это видимо глюк выгружаемой конфигурации, т.к. интерактивное удаление предопределенных принудительно устанавливает другие флаги.
Странно, как вообще такое случилось в хранилище. Буду править :)

**Comment by Pr-Mex** [comment]
В новых версиях платформы добавили контроля при загрузке из XML. Поэтому возможны варианты, когда галочки снимаются или ставятся исходя из новых зависимостей между ними.
Именно новых багов при загрузке ролей я не находил.

**Comment by karnilaev** [comment]
Спасибо. Отпишусь как будет вести себя конфигурация после того, как я сниму галки с действиями над предопределенными данными.

**Comment by karnilaev** [comment]
После того, как убрал все галки с действиями над предопределенными элементами права после разбора стали корректными. Спасибо!
